### DISCUSSION TITLE
ADP RSS feeds throwing Unknown error

### DISCUSSION THREAD

**Discussion by arseru** [FreshRSS/FreshRSS#6688] [Q&A] [answered]
### Describe the bug

Hi there,

I was trying to add the RSS feeds from ADP company (https://investors.adp.com/investor-resources/rss/default.aspx), which are the following four:

* https://investors.adp.com/rss/pressrelease.aspx
* https://investors.adp.com/rss/event.aspx
* https://investors.adp.com/rss/presentation.aspx
* https://investors.adp.com/rss/SECFiling.aspx?Exchange=CIK&Symbol=0000008670

An example of the error shown in the FreshRSS logs is:

`Unknown error for feed [https://investors.adp.com/rss/pressrelease.aspx] `

Since inoreader is able to add them without issues, I was wondering whether I can help find out what's going on inside FreshRSS.

### To Reproduce

1. Go to your FreshRSS self-hosted instance.
2. Click on the `+` button on the top left sidebar.
3. In the "Add a feed" section, add the following Feed URL: https://investors.adp.com/rss/pressrelease.aspx
4. See error: `The newsfeed could not be added. Check FreshRSS logs for details. You can try force adding by appending #force_feed to the URL.`
5. Try adding the forced feed (https://investors.adp.com/rss/pressrelease.aspx#force_feed), but the same errors appears.

### Expected behavior

The feed is added to the FreshRSS database.

### FreshRSS version

1.23.1

### Environment information

- Database version: I can't find it, but I'm using the [official Docker image](https://github.com/FreshRSS/FreshRSS/tree/edge/Docker) if it helps.
- PHP version: 8.2.7
- Installation type: [Docker image](https://github.com/FreshRSS/FreshRSS/tree/edge/Docker)
-Web server type: nginx
- Device: Lenovo ThinkPad
- OS: Arch Linux
- Browser: Firefox 128.0.3 (64-bit)

**Comment by Alkarex** [comment, marked as answer]
Seems to be yet again some feeds blocked due to Cloudflare

```sh
curl -v 'https://investors.adp.com/rss/pressrelease.aspx'
```

https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Finvestors.adp.com%2Frss%2Fpressrelease.aspx
https://github.com/FreshRSS/FreshRSS/discussions/6287
https://github.com/FreshRSS/FreshRSS/issues/4323

I just tested that it works by using a Feedburner proxy: https://feedburner.google.com/

**Comment by arseru** [comment]
Thanks a lot @Alkarex ! It indeed works for me as well using a Feedburner proxy (I never knew that existed, so double thanks) :)
